assets/src/wizards/subscriptions/index.js

```diff
@@ -30,6 +33,7 @@ class SubscriptionsWizard extends Component {
 	constructor() {
 		super( ...arguments );
 		this.state = {
+			currentTask: 0,
 			inputTextValue1: "Input value",
 			inputTextValue2: "",
 			image: null,
```

Comment:
- here in the Wizard, added a `completedTasks` Array to state, to keep track of the completed ones (it holds indexes of completed tasks -- starting from 0). In this example the **3rd** task is marked as complete, with index **2**

```suggestion
			image: null,
			completedTasks: [ 2 ],
```